### PR TITLE
React Developer Tools

### PR DESCRIPTION
**Pull request by danielkraus**
Closes https://github.com/reactjs/cs.react.dev/issues/10

### CODE REVIEW

File: src/content/learn/react-developer-tools.md [hunk 1]
```diff
@@ -1,61 +1,60 @@
 ---
-title: React Developer Tools
+title: Vývojářské nástroje Reactu
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:2]: What do you think about changing the form?
```suggestion
title: Vývojářské nástroje pro React
```

File: src/content/learn/react-developer-tools.md [hunk 2]
```diff
@@ -1,61 +1,60 @@
...
 
-Use React Developer Tools to inspect React [components](/learn/your-first-component), edit [props](/learn/passing-props-to-a-component) and [state](/learn/state-a-components-memory), and identify performance problems.
+Vývojářské nástroje Reactu slouží k prozkoumávání React [komponent](/learn/your-first-component), k úpravám [vlastností](/learn/passing-props-to-a-component) a [stavů](/learn/state-a-components-memory), a k detekování možných problémů s výkonem aplikace.
 
 </Intro>
 
 <YouWillLearn>
 
-* How to install React Developer Tools
+* Jak nainstalovat Vývojářské nástroje Reactu
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:13]: I am not sure about the capital v in "Vývojářské" and maybe we can add "si".
```suggestion
* Jak si nainstalovat vývojářské nástroje Reactu
```

File: src/content/learn/react-developer-tools.md [hunk 3]
```diff
@@ -1,61 +1,60 @@
...
 
 </YouWillLearn>
 
-## Browser extension {/*browser-extension*/}
+## Rozšíření prohlížeče {/*browser-extension*/}
 
-The easiest way to debug websites built with React is to install the React Developer Tools browser extension. It is available for several popular browsers:
+Nejjednodušší způsob, jak ladit webové stránky vytvořené pomocí Reactu, je nainstalovat si Vývojářské nástroje Reactu jako rozšíření do vašeho prohlížeče. Tyto nástroje jsou k dispozici pro následující prohlížeče:
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:19]: I think we can skip "vašeho". I am also not sure about the capital v.
```suggestion
Nejjednodušší způsob, jak ladit webové stránky vytvořené pomocí Reactu, je nainstalovat si vývojářské nástroje Reactu jako rozšíření do prohlížeče. Tyto nástroje jsou k dispozici pro následující prohlížeče:
```

File: src/content/learn/react-developer-tools.md [hunk 4]
```diff
@@ -1,61 +1,60 @@
...
 
-* [Install for **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
-* [Install for **Firefox**](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
-* [Install for **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil)
+* [Instalovat pro **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
+* [Instalovat pro **Firefox**](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
+* [Instalovat pro **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil)
 
-Now, if you visit a website **built with React,** you will see the _Components_ and _Profiler_ panels.
+Když po instalaci navštívíte jakoukoliv webovou stránku **vytvořenou v Reactu,** uvidíte panely s názvy _Components_ a _Profiler_.
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:25]: A bit different form.
```suggestion
Nyní, když navštívíte jakoukoliv webovou stránku **vytvořenou v Reactu,** uvidíte panely s názvy _Components_ a _Profiler_.
```

File: src/content/learn/react-developer-tools.md [hunk 5]
```diff
@@ -1,61 +1,60 @@
...
-For other browsers (for example, Safari), install the [`react-devtools`](https://www.npmjs.com/package/react-devtools) npm package:
+### Safari a jiné prohlížeče {/*safari-and-other-browsers*/}
+Pro další prohlížeče (jako například Safari) si nainstalujte npm balíček [`react-devtools`](https://www.npmjs.com/package/react-devtools):
 ```bash
 # Yarn
 yarn global add react-devtools
 
 # Npm
-npm install -g react-devtools
+npm install -g react-devtools/
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:36]: This is probably typo.
```suggestion
npm install -g react-devtools
```

File: src/content/learn/react-developer-tools.md [hunk 6]
```diff
@@ -64,13 +63,13 @@ yarn global add react-devtools
...
+Poté nástroje otevřte v Terminálu:
 ```bash
 react-devtools
 ```
 
-It should connect to any local React Native app that's running.
+Nástroje se nyní sami spojí s vaší React Native aplikací, která běží v lokálním prostředí.
 
-> Try reloading the app if developer tools doesn't connect after a few seconds.
+> Zkuste obnovit aplikaci pokud se nástroje po několika sekundách nepřipojí sami.
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:73]: I think, there should be y. I found this description https://www.pravopisne.cz/2015/02/samy-x-sami/.
```suggestion
> Zkuste obnovit aplikaci pokud se nástroje po několika sekundách nepřipojí samy.
```

File: src/content/learn/react-developer-tools.md [hunk 7]
```diff
@@ -64,13 +63,13 @@ yarn global add react-devtools
...
 ```
 
-Next open the developer tools from the terminal.
+Poté nástroje otevřte v Terminálu:
 ```bash
 react-devtools
 ```
 
-It should connect to any local React Native app that's running.
+Nástroje se nyní sami spojí s vaší React Native aplikací, která běží v lokálním prostředí.
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:71]: Same comment as the next one, line 73.
```suggestion
Nástroje se nyní samy spojí s vaší React Native aplikací, která běží v lokálním prostředí.
```

File: src/content/learn/react-developer-tools.md [hunk 8]
```diff
@@ -64,13 +63,13 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal.
+Poté nástroje otevřte v Terminálu:
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:66]: I think there should be no capital T.
```suggestion
Poté nástroje otevřte v terminálu:
```

File: src/content/learn/react-developer-tools.md [hunk 9]
```diff
@@ -1,61 +1,60 @@
...
+A dále je napojte do vaší stránky přídáním `<script>` tagu na začátek `<head>`:
 ```html {3}
 <html>
   <head>
     <script src="http://localhost:8097"></script>
 ```
 
-Reload your website in the browser now to view it in developer tools.
-
+Nyní obnovte stránku v prohlížeči a uvidíte Vývojářské nástroje.
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:51]:
```suggestion
Nyní obnovte stránku v prohlížeči a uvidíte vývojářské nástroje.
```

File: src/content/learn/react-developer-tools.md [hunk 10]
```diff
@@ -1,61 +1,60 @@
...
 # Yarn
 yarn global add react-devtools
 
 # Npm
-npm install -g react-devtools
+npm install -g react-devtools/
 ```
 
-Next open the developer tools from the terminal:
+Poté otevřte tyto nástroje pomocí Terminálu:
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:39]:
```suggestion
Poté otevřte tyto nástroje pomocí terminálu:
```

**Review by pchalupa — CHANGES_REQUESTED** [review]
Well done sir! I left some comments. I am not sure about capitals in the words `Terminal` and `Vývojářské nástroje`. I also added a comment about `i` vs `y. Overall this is a good translation! 👏

**Comment by danielkraus** [comment]
@pchalupa Thanks for the suggestions! 

1. I'm pretty sure Terminal should go with capital T – it's a name, see official Apple docs of Apple Czech https://support.apple.com/cs-cz/guide/terminal/apd5265185d-f365-44cb-8b09-71a064a42125/mac
2. Not that obvious with "Vyvojarske nastroje pro React" but it's a translated name that refers to a specific tool/app. If you keep "vyvojarske nastroje", that means you refer to any tool. There's also an option to keep calling it "React Developer Tools" as it's the only official way and the tool itself is not translated anyway. Similar how Google does here https://support.google.com/adsense/answer/10858959?hl=cs&sjid=3729838083219203571-EU. Thoughts?

**Comment by pchalupa** [comment]
> @pchalupa Thanks for the suggestions!
> 
> 1. I'm pretty sure Terminal should go with capital T – it's a name, see official Apple docs in Czech https://support.apple.com/cs-cz/guide/terminal/apd5265185d-f365-44cb-8b09-71a064a42125/mac
> 2. Not that obvious with "Vyvojarske nastroje pro React" but it's a translated name that refers to a specific tool/app. If you keep "vyvojarske nastroje", that means you refer to any tool. There's also an option to keep calling it "React Developer Tools" as it's the only official way and the tool itself is not translated anyway. Similar how Google does here https://support.google.com/adsense/answer/10858959?hl=cs&sjid=3729838083219203571-EU. Thoughts?

You are correct, the Terminal on Mac is with capital, but I think it is meant as a general terminal app in this context. You can use these tools on Windows, Linux, etc... and these systems have their own terminals.
I think with the "vývojářské nástroje" is similar to saying "hdyratační krém", but if you add the brand like "hydratační krém Nivea" then the capital letter is correct. I am open for discussion.

File: src/content/learn/react-developer-tools.md [hunk 11]
```diff
@@ -4,30 +4,30 @@ title: React Developer Tools
 
 <Intro>
 
-Use React Developer Tools to inspect React [components](/learn/your-first-component), edit [props](/learn/passing-props-to-a-component) and [state](/learn/state-a-components-memory), and identify performance problems.
+React Developer Tools (vývojářské nástroje pro React) slouží k prozkoumávání React [komponent](/learn/your-first-component), k úpravám [vlastností](/learn/passing-props-to-a-component) a [stavů](/learn/state-a-components-memory), a k detekování možných problémů s výkonem aplikace.
 
 </Intro>
 
 <YouWillLearn>
```
Comment (pchalupa) [inline src/content/learn/react-developer-tools.md:11]: Could you please check this component? The title is in en.

**Review by pchalupa — APPROVED** [review]

**Pull request merged**